app/main.py

```diff
@@ -1,3 +1,41 @@
-def shop_trip():
-    # write your code here
-    pass
+from _datetime import datetime
+import json
+
+from app.trips_cost import calculate_trips_cost, Shop
+
+
+def shop_trip() -> None:
+    with open("config.json", "r") as file:
+        config = json.load(file)
```

Comment:
why dont you remove all this logic from context manager? i mean, it seems you just need to get the dict from json file into config variable and thats it, all other operations can work with stored data without open context manager.

Reply:
Good point